test/conformance/ptp/ptp.go

```diff
@@ -992,3 +1055,248 @@ func printInterface(config ptpv1.PtpConfig, interfaceDetailsMap map[string]strin
 		}
 	}
 }
+
+func rebootSlaveNode(fullConfig testconfig.TestConfig) {
+	logrus.Info("Rebooting system starts ..............")
+
+	const (
+		daemonSetName    = "ptp-reboot"
+		containerName    = "container-00"
+		imageWithVersion = "quay.io/testnetworkfunction/debug-partner:latest"
+	)
+
+	// 1. create a daemon set for the node reboot
+	dsRunningPods, err := daemonsets.CreateDaemonSet(daemonSetName, PtpLinuxDaemonNamespace, containerName, imageWithVersion, timeoutIn5Minutes)
```

Comment:
Deploy the daemonset in default namespace

Reply:
Can we add better naming . ptp Linux daemonset and this is  confusing

Reply:
Sorry, I did not understand your comment. The namespace variable is changed, so we are not using PtpLinuxDaemonNamespace anymore. Is that what you suggested?

Reply:
we are using very generic name daemonset. There are two daemonset now here. ptpLinux daemonset and test harness daemonset . Hence the confusion

Reply:
@aneeshkp This is the reboot daemonset. I can change to rebootDaemonset.